### PR TITLE
Update paritychecktuning.txt

### PR DESCRIPTION
**Pull request by Core-i99**
This should be done now

### CODE REVIEW

File: Parity Check Tuning/paritychecktuning.txt [hunk 1]
```diff
@@ -134,43 +134,43 @@ will be started=zal worden gestart
...
-Following drives overheated=
-Only able to start a Read-Check due to number of disabled drives=
+%s Cancelled=%s Geannuleerd
+Array not started so no action taken=Array niet gestart dus geen actie ondernomen
+as drives now cooled down=aangezien schijven nu afgekoeld zijn
+Following drives overheated=Volgende schijven oververhit
+Only able to start a Read-Check due to number of disabled drives=Alleen in staat om een Read-Check te starten vanwege het aantal disabled drives
 
 ; Syslog messages (with debug logging enabled)
 ; (probably not worth translating these as they are mainly of use to the plugin developer)
```
Comment (Core-i99) [inline Parity Check Tuning/paritychecktuning.txt:176]: Kept in English as stated here

**Comment by Core-i99** [comment]
Certain parts are kept in English because that's stated in the translation file.

Kept in English:
- Help text on settings page //These could perhaps be left in English to keep translation task easier)
- Syslog messages (with debug logging enabled) //; (probably not worth translating these as they are mainly of use to the plugin developer)

**Comment by spencerjunraid** [comment]
Ok to merge @bergware?

**Comment by bergware** [comment]
Yes, looks fine.
Thanks

**Pull request merged**